### DISCUSSION TITLE
Solidstart project deployed on vercel got 404

### DISCUSSION THREAD

**Discussion by LavaCxx** [solidjs/solid-start#1444] [Q&A] [answered]
I found this problem from my own project, and later tried the official basic and hackernews templates, but it was still 404.
<img width="570" alt="image" src="https://github.com/solidjs/solid-start/assets/48022591/5edd452c-81d3-4a0f-841b-29b089f9b75e">
<img width="948" alt="image" src="https://github.com/solidjs/solid-start/assets/48022591/61012ac9-0d36-4050-8f57-308c36cc39c4">

**Comment by cleggacus** [comment, marked as answer]
I dont know if this is the same issue I had but I had a similar thing but it turned out i just needed to specifiy vercel as the server preset in the config. 

```ts
export default defineConfig({
    server: {
        preset: "vercel"
    }
});
```

**Reply by amitsingh-007** [reply]
This worked...thanks
